ios/scripts/asc_assign_external_testflight_group.py

```
@@ -253,6 +320,169 @@ def _assign_build(token: str, group_id: str, build_id: str) -> None:
+                    if review_state in ACTIVE_BETA_REVIEW_STATES:
+                        print(
+                            "asc_assign_external_testflight_group: build "
+                            f"{build_number} external state is READY_FOR_BETA_SUBMISSION with submission "
+                            f"{submission['id']} (state={review_state or 'unknown'})"
+                        )
+                        return
+                    raise RuntimeError(
+                        "build "
+                        f"{build_number} external state is READY_FOR_BETA_SUBMISSION with beta app review "
+                        f"submission {submission['id']} in unexpected betaReviewState={review_state or '<empty>'}"
+                    )
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)